packages/flutter/test/rendering/editable_test.dart

```diff
@@ -1632,7 +1632,14 @@ void main() {
         children: renderBoxes,
```

Comment:
Can you write a test to verify the fix? I imagine editable text with widgetspan will be affect by this change

Reply:
I added line 1636. It does fail without this patch. I'll add a few more for intrinsic heights as well.